builtin/logical/transit/path_encrypt.go

```diff
@@ -27,36 +44,65 @@ func (b *backend) pathEncrypt() *framework.Path {
 
 			"context": &framework.FieldSchema{
 				Type:        framework.TypeString,
-				Description: "Context for key derivation. Required for derived keys.",
+				Description: "Base64 encoded context for key derivation. Required for derived keys.",
 			},
 
 			"nonce": &framework.FieldSchema{
 				Type:        framework.TypeString,
-				Description: "Nonce for when convergent encryption is used",
+				Description: "Base64 encoded nonce for when convergent encryption is used",
```

Comment:
Can you change this to "...for when v1 convergent encryption is used"? Just to clarify.